### ISSUE TITLE
UnauthorizedAccessException. Try to save removing the savegame

### ISSUE THREAD

**Issue by Kastuk** [loiste-interactive/Obenseuer-Issues#567]
After game start (savegame from #558) I did tried to enter Tenement and got endless loading, then error in log and disappear of Autosave folder.
[Log1 Autosave removing savegame.log](https://github.com/loiste-interactive/Obenseuer-Issues/files/12775620/Log1.Autosave.removing.savegame.log)

After game restart I got several changing of scenes without issues, Did use Bepinex with UnityExplorer to deactivate some of Market objects for testing.
Then I did tested duplicating garbage bug #558 by Moving things in Tenement, then exit and enter back to see they can dublicate no more than one time.
Then I tried manual saving and got same error with disappear of this savegame folder.
I have Bepinex output log in runtime there.
![2023-10-01 00_51_04-Greenshot](https://github.com/loiste-interactive/Obenseuer-Issues/assets/36314945/ccfe3c55-aff1-45eb-a047-889eea86aaa4)
[Log 2 Last autosave before error.zip](https://github.com/loiste-interactive/Obenseuer-Issues/files/12775617/Log.2.Autosave.before.error.zip)
[Log2 Manual saving removing savegame.log](https://github.com/loiste-interactive/Obenseuer-Issues/files/12775619/Log2.Manual.saving.removing.savegame.log)

**Comment by Oskutin** [comment]
Some other application is using the file, and or modding tools could interfere it

**Comment by Kastuk** [comment]
I got same error after playing normal for hours without any third-party modding. Then I quit game, start again and load that navigation-challenged savegame:
[Heavy methanol blinding in tight sauna.zip](https://github.com/loiste-interactive/Obenseuer-Issues/files/12786761/Heavy.methanol.blinding.in.tight.sauna.zip)
Did fall asleep under pills there, then detoxicated and move around Dekula A, going to General store to find an food and got this endless loading.
![20231003002556_1](https://github.com/loiste-interactive/Obenseuer-Issues/assets/36314945/80ad95a3-3d2d-4050-bdb7-233e32f44431)
Loading screen still here, but player already active and start taking damage under hunger.
I did manage to load menu scene to reload savegame, but autosave disappeared again
[loading screen not deactivated after load.log](https://github.com/loiste-interactive/Obenseuer-Issues/files/12786753/loading.screen.not.deactivated.after.load.log)
